readme.md

```diff
@@ -4,26 +4,103 @@
+- [OpenFaas](https://www.openfaas.com/) - Serverless functions framework.
+
+### Version control and releasing
+- [Semantic Versioning](https://semver.org/) - Versioning specification.
+- [semantic-release](https://github.com/semantic-release/semantic-release) - Fully automated package publishing.
+- [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) - Lightweight specification for commit messages.
+- [gitmoji](https://gitmoji.dev/) - Emoji-based commit message convention
+
+### Version management
 - [NVM](https://github.com/nvm-sh/nvm) - Node Version Manager - POSIX-compliant bash script to manage multiple active Node.js versions.
-- [Snyk.io](https://snyk.io) - Automatically find, prioritize and fix vulnerabilities in the open source dependencies used to build your cloud native applications.
-- [Tabnine](https://www.tabnine.com/) - Tabnine's powerful Artificial Intelligence assistant works right where you are, in the comfort of your IDE.
 - [rbenv](https://github.com/rbenv/rbenv) - Use rbenv to pick a Ruby version for your application and guarantee that your development environment matches production.
-- [WhiteSource Renovate](https://www.whitesourcesoftware.com/free-developer-tools/renovate) - Save time and reduce risk by automating dependency updates in software projects.
+- [asdf](https://asdf-vm.com/) - Version management for multiple frameworks.
 
+### Terminal
+- [Oh My Zsh](https://ohmyz.sh/) - Oh My Zsh is a delightful, open source, community-driven framework for managing your Zsh configuration. It comes bundled with thousands of helpful functions, helpers, plugins, themes, and a few things that make you shout.
+- [Fig](https://fig.io/) - Auto-completion for terminal
+
+### Local development
+- [LocalStack](https://github.com/localstack/localstack) - Local emulator of AWS services.
+- [ngrok](https://ngrok.com/) - Locally available reverse proxy
+- [Tabnine](https://www.tabnine.com/) - Tabnine's powerful Artificial Intelligence assistant works right where you are, in the comfort of your IDE.
+- [Github Copilot](https://copilot.github.com/) - AI code completion
 ### SaaS
 
 - [DX Scanner Dashboard](https://dxscanner.io) - Measure, analyze, increase and validate your team's effectiveness with all detailed data in one place. Discover data that help you facilitate decision making.
 - [Fossa](https://fossa.com/) - The most complete open source management and policy engine for enterprise security and compliance.
+- [SonarQube](https://www.sonarqube.org/) - Static code quality analysis.
+- [deepsource](https://deepsource.io/) - Static code analysis.
+- [Fly](https://fly.io/) - Full stack apps and database hosting.
+- [LinearB](https://linearb.io/) - Software delivery intelligence.
+- [Pluralsight Flow](https://www.pluralsight.com/product/flow) - Project workflow dashboard.
+- [Gitpod](https://www.gitpod.io/) - Instant IDE.
+- [MinIO](https://min.io/) - S3-compatible object storage.
```

Comment:
https://github.com/minio/minio

this would be better to move to open-source with given link cause it's usually used as a docker image in testing pipeline